csharp/src/Google.Protobuf.Test/UnknownFieldSetTest.cs

```diff
@@ -138,6 +138,30 @@ public void TestClone(IMessage message)
             Assert.AreEqual(message.ToByteArray(), otherEmptyMessage.ToByteArray());
         }
 
+        [Test]
+        public void TestClone_lengthDelimited()
```

Comment:
nit: rename to TestClone_LengthDelimited()

Reply:
Also, from the name it's not fully clear what this is actually supposed to test.

Reply:
done